Packs/CortexXpanse/Integrations/CortexXpanse/README.md

```diff
@@ -0,0 +1,1432 @@
+
+
+### asm-list-alerts
+***
+Get a list of all your ASM alerts filtered by alert IDs, severity and/or creation time. Can also sort by creation time or severity. Maximum result limit is 100 assets.
+
+
+#### Base Command
+
+`asm-list-alerts`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| alert_id_list | Comma separated list of integers of the alert ID. | Optional | 
+| severity | Comma separated list of strings of alert severity (valid values are low, medium, high, critical, informational). | Optional | 
+| lte_creation_time | A date in the format 2019-12-31T23:59:00. Only incidents that were created on or before the specified date/time will be retrieved. | Optional | 
+| gte_creation_time | A date in the format 2019-12-31T23:59:00. Only incidents that were created on or after the specified date/time will be retrieved. | Optional | 
+| sort_by_creation_time | Sorts returned incidents by the date/time that the incident was created ("asc" - ascending, "desc" - descending). Possible values are: asc, desc. | Optional | 
+| sort_by_severity | Sorts returned incidents by the date/time that the incident was created ("asc" - ascending, "desc" - descending). Possible values are: asc, desc. | Optional | 
```

Comment:
I think this should be:
Sorts returned incidents by severity of the incident.

Reply:
they are alerts in XSIAM/XpanseV2